src/actions/keyBasedAssetTransferActions.js

```diff
@@ -243,9 +243,14 @@ export const calculateKeyBasedAssetsToTransferTransactionGasAction = () => {
     if (isCalculatingGas) return;
     dispatch({ type: SET_CALCULATING_KEY_BASED_ASSETS_TO_TRANSFER_GAS, payload: true });
 
-    await dispatch(fetchGasInfoAction());
+    await dispatch(fetchGasInfoAction(CHAIN.ETHEREUM));
     const { history: { gasInfo } } = getState();
-    const gasPrice = getGasPriceWei(gasInfo);
+    if (!gasInfo.isFetched || !gasInfo.gasPrice) {
+      reportLog('calculateKeyBasedAssetsToTransferTransactionGasAction failed: no gas price.');
+      return;
+    }
+
+    const gasPrice = gasInfo.gasPrice.instant;
```

Comment:
Why instant? Would it make sense to use `fast` as more cost effective solution?

Reply:
We had `max` before so just replaced to whatever is fastest is coming from `Etherspot` gas prices. Hope it makes sense.